browserify-typescript/index.js

```diff
@@ -31,13 +33,25 @@ var defaultOptions = {
   onLog: function(log){
     console.log((log = log.split(' '), log[0] = pretty(log[0]), log.join(' ')));
   }
-}
+};
 
 module.exports = function(options) {
   options = merge(defaultOptions, options);
 
   var b = browserify(options.src, options.browserifyOptions)
-    .plugin(tsify, options.tsifyOptions);
+    .plugin(tsify, options.tsifyOptions)
+      .transform(function (file) {
+          return through(function (buf, enc, next) {
+              ng2TemplateParser({contents: buf, path: file}, {
```

Comment:
Maybe you can inherit `options`? That way ng2TemplateParser can use `target` or we can change `base` options